code/modules/mob/living/silicon/pai/software_modules_vr.dm

```diff
@@ -0,0 +1,35 @@
+/datum/pai_software/translator
+	name = "Universal Translator"
+	ram_cost = 35
+	id = "translator"
+
+	toggle(mob/living/silicon/pai/user)
```

Comment:
just override /datum/pai_software/translator/toggle(...), no need to override the whole thing